package ru.unlimit.javapro.lab29.exceptions;

import java.io.FileInputStream;
import java.io.FileNotFoundException;

public class ExceptionApp {

	public static void main(String[] args) {

		
//		int a = 5;
//		int b = 0;
//		int c = a/b;


//		try {
//			FileInputStream fis = new FileInputStream("C:/ALL/tmp/123.txt");
//		} catch (FileNotFoundException e) {
//			// TODO Auto-generated catch block
//			e.printStackTrace();
//		}
//		FileNotFoundException ex;
//		
//		
//		A ref = null;
//		System.out.println(ref.i);

		
		int arr[] = new int[2];
		arr[0]=3;
		arr[1]=4;
		arr[2]=5;
		ArrayIndexOutOfBoundsException ex;
	}
}
class A{
	int i;
}
